src/test/java/duke/command/AddCommandTest.java

```diff
@@ -0,0 +1,58 @@
+package duke.command;
+
+import java.time.format.DateTimeParseException;
+
+import org.junit.jupiter.api.Test;
+
+import duke.DukeException;
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class AddCommandTest {
+    @Test
+    public void AddSingleTodo() throws DukeException {
```

Comment:
Perhaps a good way to go forward would be to follow the naming convention for tests! For example, 

```
unitBeingTested_descriptionOfTestInputs_expectedOutcome
```